docs/pom.xml

```diff
@@ -40,11 +40,96 @@
                         </goals>
                         <configuration>
                             <target>
-                                <exec dir="${project.basedir}" executable="jekyll" failonerror="true">
+                                <available file="${project.build.directory}/che-repo" property="cheRepoExists" type="dir"/>
+                                <mkdir dir="${project.build.directory}/che-repo"/>
+                                <exec dir="${project.basedir}" executable="docker" failonerror="true">
+                                    <arg value="run" />
+                                    <arg value="--name" />
+                                    <arg value="jekyll_git_che" />
+                                    <arg value="jekyll/jekyll" />
+                                    <arg value="sh" />
+                                    <arg value="-c" />
+                                    <arg value="cd /srv/jekyll/ ; git init ; git remote add -t add-docs-images -f origin https://github.com/eclipse/che.git ; git checkout add-docs-images" />
```

Comment:
yes but as said it would be a lot more compliant to use maven artifacts instead of doing some clone operations.  
It simplify dependencies as it's a feature natively provided by maven